pwnlib/commandline/pwnstrip.py

```diff
@@ -32,23 +34,23 @@ def main(args):
         for offset in pwnlib.libcdb.get_build_id_offsets():
             data = elf.read(elf.address + offset + 0xC, 4)
             if data == 'GNU\x00':
-                elf.write(elf.address + offset + 0x10, read('/dev/urandom', 20))
+                elf.write(elf.address + offset + 0x10,
+                          read('/dev/urandom', 20))
 
     for function in args.patch:
         if function not in elf.symbols:
             log.error("Could not find function %r" % function)
-
-        trap = asm(shellcraft.trap())
         offset = elf.symbols[function]
-
-        elf.write(elf.address + offset, trap)
+        elf.asm(offset, 'ret')
 
     result = elf.data
 
-    if args.output.isatty():
-        result = enhex(result)
+    if args.output:
+        elf.save(args.output)
+        os.chmod(args.output, 0o755)
```

Comment:
This needs to check if `args.output` is a file (`os.path.isfile`) OR needs to catch any exceptions that might get raised.